app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def posts; end
+end
```

Comment:
- [ ] I will suggest you add `index` and `show` controller actions for the post as well.

Reply:
Thanks again!